.travis.yml

```diff
@@ -30,3 +32,5 @@ script:
   - if [ "$DIMAGE" = graemeastewart/u16-dev ]; then docker run -e CXX=g++-7 -e CC=gcc-7 -v $(pwd):/mnt $DIMAGE /mnt/.travis-scripts/build-test.sh; fi
```

Comment:
I think we can drop u16 as per our discussion at #122.

Reply:
Done in latest commits